core/services/keeper/registry_interface.go

```diff
@@ -56,10 +57,10 @@ type RegistryWrapper struct {
 	contract1_1 *registry1_1.KeeperRegistry
 	contract1_2 *registry1_2.KeeperRegistry
 	contract1_3 *registry1_3.KeeperRegistry
-	evmClient   bind.ContractBackend
+	backend     evmclient.Client
```

Comment:
I think it is more cleaner to just use the name evmClient instead of backend in both the variables, since we no longer use ContractBackend type.